Agreements/Internal/TCCPT_TCVAT.txt

```diff
@@ -0,0 +1,7 @@
+COPX:*:*:OCK:EGSS:*:London TC CPT:London TC VATON:*:18000:OCK
+COPX:*:*:OCK:EGSC:*:London TC CPT:London TC VATON:*:18000:OCK
+COPX:*:*:OCK:EGGW:*:London TC CPT:London TC VATON:*:18000:OCK
+
+COPX:*:*:BIG:EGLC:*:London TC CPT:London TC VATON:*:18000:BIG
+COPX:*:*:BIG:EGKB:*:London TC CPT:London TC VATON:*:18000:BIG
+COPX:*:*:BIG:EGTO:*:London TC CPT:London TC VATON:*:18000:BIG
```

Comment:
This traffic is 'skipped' by TC CPT, straight to TC VATON - so can be removed here

Reply:
Should we not point the previous agreement to VATON and make VATON a guest in TC CPT?